build_run_test.sh

```diff
@@ -8,17 +8,34 @@ docker build --tag $NAME \
              --cache-from $REPO \
              context
 
-DATA_DIR=/tmp/docker_igv_js_`date +"%Y-%m-%d_%H-%M-%S"`
-cp -a input_fixtures $DATA_DIR
-# The on_startup script writes to the shared volume,
-# so we copy first to avoid contamination of the fixture in source.
+PYTHON_SERVER_PORT=9999
+
+# https://stackoverflow.com/a/13322667
+# Finds the correct interface and ip, based on the machine's default route.
+# Returns a url that will resolve to the SimpleHTTPServer we spin up below 
+# so that the Docker container can make requests to files existing 
+# under the cwd i.e. `./test-data/input.json`
+get_python_server_url() {
```

Comment:
@mccalluc I feel badly about using this again, but it works...

Reply:
We can't do this. it might be catching edge cases we're unaware of, but if it did break, we're not in a place to understand it's behavior. There's a grep example earlier in that thread, or I could find perl readable here:
```
ifconfig | perl -ne 'print $2 if /inet (addr:)?(\d+\.\d+\.\d+\.\d+)/ and $2 ne "127.0.0.1"'
```

Reply:
Or move this whole script into python?